source/taup/intro.rst

```diff
@@ -0,0 +1,40 @@
+简介
```

Comment:
Please rewrite the software metadata information following [the template](https://github.com/seismo-learn/software/pull/27).